src/NuGet.Clients/PackageManagement.VisualStudio/ProjectSystems/BuildIntegratedProjectSystem.cs

```diff
@@ -298,10 +299,14 @@ private IScriptExecutor ScriptExecutor
                         if (pathToProject.TryGetValue(xProjPath, out xProjDTE))
                         {
                             var xProjFullPath = EnvDTEProjectUtility.GetFullProjectPath(xProjDTE);
-                            childReferences.Add(xProjFullPath);
 
-                            // Continue walking this project if it has not been walked already
-                            result.ToProcess.Add(new DTEReference(xProjDTE, xProjFullPath));
+                            if (!string.IsNullOrEmpty(xProjFullPath))
+                            {
```

Comment:
Looks like this could end up being used in some situations, I'm going to leave this in as an extra safety check since it could return null here.